[docs] clarify depends docs (#6200)
sveltejs/kit: documentation/docs/05-load.md

#### depends

> This function declares that the `load` function has a _dependency_ on a variable number of specific URLs, which can subsequently be used with [`invalidate()`](/docs/modules#$app-navigation-invalidate) to cause `load` to rerun.

Reviewer: ```suggestion
This function declares that the `load` function has a _dependency_ on one or more URLs, which can subsequently be used with [`invalidate()`](/docs/modules#$app-navigation-invalidate) to cause `load` to rerun.
```
